components/Pagination/Pagination.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { number, string } from 'prop-types';
+import classNames from 'classnames';
+import LeftAngleIcon from 'static/images/icons/FontAwesome/angle-left-solid.svg';
+import RightAngleIcon from 'static/images/icons/FontAwesome/angle-right-solid.svg';
+import PaginationItem from './PaginationItem/PaginationItem';
+import styles from './Pagination.css';
+
+Pagination.propTypes = {
+  className: string,
+  currentPage: number.isRequired,
+  totalPages: number.isRequired,
+};
+
+Pagination.defaultProps = {
+  className: undefined,
+};
+
+// eslint-disable-next-line react/prop-types
+const PaginationItems = ({ currentPage, totalPages }) => {
+  const shouldTruncateStart = currentPage - 1 > 5;
+  const shouldTruncateEnd = totalPages - currentPage > 5;
+
+  const paginationStart = shouldTruncateStart ? currentPage - 4 : 1;
+  const paginationEnd = shouldTruncateEnd ? currentPage + 4 : totalPages;
+
+  const paginationLength = paginationEnd - paginationStart + 1;
+
+  const PaginationItemArray = [...Array(paginationLength)].map((_, index) => {
+    const page = index + paginationStart;
+    const isCurrent = page === currentPage;
+
+    return <PaginationItem key={page} value={`${page}`} isCurrent={isCurrent} testId={`${page}`} />;
+  });
+
+  return (
+    <>
+      {shouldTruncateStart && (
+        <>
+          <PaginationItem key="1" value="1" testId="1" />
+          <PaginationItem
```

Comment:
Can we just style an ellipse and render it here instead of forcing it into the PaginationItem component API? If you do that, you can drop `isClickable`

Reply:
I used the `PaginationItem` component for ellipsis so it would retain visual consistency. So for example if we would change CSS properties like font-size, background color etc. of `PaginationItem` it would be applied on the ellipsis as well.

Reply:
Right. I figured that, but was thinking about ways to simplify that component. I think we can just leave this alone though.